types/index.d.ts

```diff
@@ -1211,3 +1216,7 @@ declare namespace Fig {
     cache?: Cache;
   }
 }
+
+interface SymbolConstructor {
+  readonly generatorTag: unique symbol;
```

Comment:
An alternative name may be this one:
```suggestion
  readonly generatorName: unique symbol;
```